projects/editor/components/editor/editor.component.ts

```diff
@@ -170,6 +180,9 @@ export class TuiEditor extends TuiControl<string> implements OnDestroy {
     @Input()
     public tools = this.options.tools;
 
+    @Input()
+    public dropdownHandler?: (editor: TuiEditor) => boolean;
```

Comment:
Could we reuse [DropdownOpen](https://taiga-ui.dev/directives/dropdown-open) ?
Or use the same naming and similar logic ?

Reply:
fixed